packages/dso-toolkit/src/components/form/form-groups/files/form-group-files.scss

```diff
@@ -11,6 +11,19 @@
 @use "../../form.mixins" as css-form-mixins;
 
 .form-group.dso-files {
+  // if file list is empty
+  &:not(:has(.dso-filelist li)) {
```

Comment:
Ah, je hebt mijn favoriete selector weer van stal gehaald 😄

Reply:
In dit geval volkomen terecht 😎